package.json

```diff
@@ -17,6 +18,8 @@
     }
   },
   "devDependencies": {
+    "@lokalise/eslint-config-lokalise-frontend": "0.3.0",
+    "jest": "29.4.2",
```

Comment:
should we consider going with vitest instead of jest here, if we are already using vite? considering that it mostly replicates jest API, should be very smooth transition

Reply:
jest being here is a bit of a red herring for this PR.

jest isn't actually used directly by anything in this PR, but we require its presence in the package.json so that the lint config can find the jest version to use. 

There'll be a later PR to add a dummy test and a test script. I'll try out Vitest on that one... which might mean rethinking the "jest" part of the linting 😅